.ci/azure/linux_arm64.yml

```diff
@@ -17,13 +21,31 @@ jobs:
     system.debug: true
     VSTS_HTTP_RETRY: 5
     VSTS_HTTP_TIMEOUT: 200
+    PYTHON_ARM_VERSION: "3.8.12"
+    OPENVINO_ARCH: 'aarch64'
+    NUM_PROC: 1
     BUILD_TYPE: Release
-    REPO_DIR: $(Build.Repository.LocalPath)
-    OPENVINO_REPO_DIR: $(REPO_DIR)/../openvino
-    OPENCV_REPO_DIR: $(REPO_DIR)/../opencv
+    REPO_OPENVINO: $(REPO_CONTRIB)/../openvino
```

Comment:
should be like
    OPENVINO_REPO_DIR: $(Build.Repository.LocalPath)
    OPENVINO_CONTRIB_REPO_DIR: $(OPENVINO_REPO_DIR)/../openvino_contrib
    OPENCV_REPO_DIR: $(OPENVINO_REPO_DIR)/../opencv

Reply:
done